src/pulselistener/pulselistener/lib/mercurial.py

```diff
@@ -15,9 +15,11 @@
 
 import hglib
 import structlog
+from libmozdata.phabricator import revision_available
```

Comment:
I'd reimplement `revision_available` here (it's just a few lines), so we don't have to import phabricator here